cleo_sdk/CLEO.h

```diff
@@ -377,6 +414,8 @@ struct CRunningScript
 	void SetNotFlag(bool state) { NotFlag = state; }
 
 	eDataType PeekDataType() const { return *(eDataType*)CurrentIP; }
+	eArrayDataType PeekArrayDataType() const { return *(eArrayDataType*)(CurrentIP + 1 + 2 + 2 + 1); } // valid only for params of array type
```

Comment:
first 7 bits define the type, 8th bit control it the index is local or global
https://gtamods.com/wiki/SCM_Instruction#Arrays

Reply:
I tested it a bit and seems the local/global type describes INDEXER variable type, not (as expected) the array type.